src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/Listing.html

```diff
@@ -1,3 +1,6 @@
+<div class="overview-description">
+    <adh-parse-markdown data-parsetext="overviewDesc"></adh-parse-markdown>
+</div>
```

Comment:
IMHO this should not be part of the listing itself. It should be on top of the listing instead (ResourceArea.html)